.github/workflows/publish-to-pypi.yml

```diff
@@ -47,7 +47,7 @@ jobs:
 
     - name: Build source and wheel distributions
       run: |
-        python setup.py sdist bdist_wheel
+        make package
```

Comment:
I almost want to keep this command as a pure python command, that way people without knowledge of `make` (e.g. Windows users) can understand what is happening as well.

Reply:
We're heavily using `make` commands in our workflows. The reason for replacing `python setup.py` with `make` is to keep it simple when we switch to other tools for building packages.

Reply:
Let's not jinx things, I'm hoping Github Actions will last for a couple more years at least! Or maybe you mean switching to use `poetry` or something? EIther way, ok with using `make` here even though it's biased towards Linux/macOS.

Reply:
I meant switching to `build` (https://github.com/pypa/build) or poetry.